### PR TITLE
Convert all config file line endings to unix style

### PR DESCRIPTION
**Pull request by optimumtact**
I need to clarify this but by default file2list looks for /n not /r/n

### CODE REVIEW

**Comment by optimumtact** [comment]
https://github.com/tgstation/-tg-station/pull/10572/files?w=1 for whitespace supressed diff changes

File: config/access levels.txt [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
-	access_captain = 23
-	access_all_personal_lockers = 24
-	access_chapel_office = 25
-	access_tech_storage = 26
-	access_atmospherics = 27
-	access_bar = 28
-	access_janitor = 29
-	access_disposal_units = 30
-	access_hydroponics = 35
+HOW TO CONVERT A MAP TO THE NEW (june 2008) ACCESS LEVEL SYSTEM
```
Comment (Cheridan) [inline config/access levels.txt]: >new 
>june 2008

vital info right hurr

Comment (Jalleo) [inline config/access levels.txt]: Old commented out info in repository?

Comment (optimumtact) [inline config/access levels.txt]: Just delete this file?

**Comment by optimumtact** [comment]
Ready to merge

**Comment by optimumtact** [comment]
ready to merge

**Comment by optimumtact** [comment]
@Cheridan pls

**Pull request merged**